process_email.py

```diff
@@ -609,11 +609,13 @@ def _handle_attachment(self, part, file_name, file_path):
                 curr_attach['should_ignore'] = True
 
         part_payload = part.get_payload(decode=True)
-        if not part_payload:
-            return phantom.APP_SUCCESS
         try:
-            with open(file_path, 'wb') as f:
-                f.write(part_payload)
+            if not part_payload:
+                _ = open(file_path, 'wb')
+                self._base_connector.debug_print("Payload not found, hence adding empty file")
```

Comment:
can we use `with` with the `open` here to auto close the file after writing to it?

```python
with open(file_path, 'wb') as _:
  self._base_connector.debug_print("Payload not found, hence adding empty file")
```